src/routes/hall.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+import HallControllers from '../controllers/hall.controllers';
+import Container from 'typedi';
+
+const router = Router();
+const hallController = Container.get(HallControllers);
+
+router.route('/').post(hallController.createHall).get(hallController.getAllHalls);
+
+router
+  .route('/:id')
```

Comment:
what is this? please look at it properly. look at other routers i created.

Reply:
Since I needed just the two url, that's why I used it that way, it works

Reply:
okay. but the goal is to make things understandable and clear, as long as it's not a personal project, other people might pick it up in the future.